Lib/bdb.py

```diff
@@ -548,6 +554,10 @@ def _set_stopinfo(self, stopframe, returnframe, stoplineno=0, opcode=False):
         # stoplineno >= 0 means: stop at line >= the stoplineno
         # stoplineno -1 means: don't stop at all
         self.stoplineno = stoplineno
+        # startframe/startlineno is the frame/line number when the user does
+        # step or next. We don't want to stop at the same line for those commands.
+        self.startframe = startframe
```

Comment:
Maybe we could pick a more informative name, something like "command frame", because it's ambiguous here what exactly started at that frame.